products/red-hat-openshift.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Red Hat OpenShift
+category: server-app
+iconSlug: redhatopenshift
+permalink: /red-hat-openshift
+alternate_urls:
+-   /openshift
+-   /rh-openshift
+versionCommand: oc version
+releasePolicyLink: https://access.redhat.com/support/policy/updates/openshift
+releaseImage: https://access.redhat.com/sites/default/files/styles/XL%20-%20Extra%20Large/public/images/ocp_lifecycle_eus_v5.png
+changelogTemplate: https://docs.openshift.com/container-platform/__RELEASE_CYCLE__/release_notes/ocp-{{"__RELEASE_CYCLE__"| replace:'.','-'}}-release-notes.html
+releaseDateColumn: true
+activeSupportColumn: true
+eolColumn: Maintenance Support
+extendedSupportColumn: true
+
+releases:
+-   releaseCycle: "4.12"
+    releaseDate: 2023-01-17
+    support: true
+    eol: 2024-07-17
+    latest: "4.12.6"
+    latestReleaseDate: 2023-03-07
+    extendedSupport: 2025-01-17
+
+-   releaseCycle: "4.11"
+    releaseDate: 2022-08-10
+    support: 2023-04-17
+    eol: 2024-02-10
+    latest: "4.11.30"
+    latestReleaseDate: 2023-03-07
+    extendedSupport: false
+
+-   releaseCycle: "4.10"
```

Comment:
According to the product description this release should have Extended Support.

Reply:
But that's not documented on https://access.redhat.com/support/policy/updates/openshift...